src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -90,11 +94,11 @@
     public ResponseEntity<CallbackResponse> generateGrant(@RequestBody CallbackRequest callbackRequest)
             throws NotificationClientException {
 
-        List<Document> documents = new ArrayList<>();
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         @Valid CaseData caseData = caseDetails.getData();
         DocumentType template;
         Document digitalGrantDocument;
+        getRegistryDetails(callbackRequest);
```

Comment:
You could pass caseDetails as a parameter to getRegistries since we already have it.

Reply:
Committed change to use caseDetails as opposed to callBackRequest